_includes/pbc/all/install-features/202307.0/install-the-dynamic-store-prices-feature.md

```diff
@@ -0,0 +1,517 @@
+          namespace="Orm\Zed\Currency\Persistence" package="src.Orm.Zed.Currency.Persistence">
+
+    <table name="spy_currency_store" idMethod="native">
+        <behavior name="event">
+            <parameter name="spy_currency_store_all" column="*"/>
+        </behavior>
+    </table>
+
+</database>
+
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied by adding a currency to the store in the Backoffice interface. 
+
+{% endinfo_block %}
+
+2. Run the following commands to apply database changes and generate entity and transfer changes:
+
+```bash
+console propel:install
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied by checking your database:
+
+| DATABASE ENTITY                       | TYPE   | EVENT   |
+|---------------------------------------|--------|---------|
+| spy_store.fk_currency                 | column | added   |
+| spy_currency_store                    | table  | added   |
+
+{% endinfo_block %}
+
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied in transfer objects:
+
+| TRANSFER | TYPE | EVENT | PATH |
+| --- | --- | --- | --- |
+| CurrencyCriteria | class | created | src/Generated/Shared/Transfer/CurrencyCriteriaTransfer  |
+| Store.defaultCurrencyIsoCode     | property | added | src/Generated/Shared/Transfer/CustomerTransfer  |
+
+{% endinfo_block %}
+
+### 3) Configure export to Redis
+
+1.  Set up publisher plugins:
+
+| PLUGIN | SPECIFICATION | PRERQUISITES | NAMESPACE |
+| --- | --- | --- | --- |
+| CurrencyStoreWritePublisherPlugin | Publishes currency store data to storage table. | None | Spryker\Zed\StoreStorage\Communication\Plugin\Publisher\CurrencyStore |
+
+
+**src/Pyz/Zed/Publisher/PublisherDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Publisher;
+
+use Spryker\Zed\Publisher\PublisherDependencyProvider as SprykerPublisherDependencyProvider;
+use Spryker\Zed\StoreStorage\Communication\Plugin\Publisher\CurrencyStore\CurrencyStoreWritePublisherPlugin;
+
+class PublisherDependencyProvider extends SprykerPublisherDependencyProvider
+{
+    protected function getPublisherPlugins(): array
+    {
+        return array_merge(
+            // ...
+            $this->getStoreStoragePlugins(),
+        );
+    }
+    
+    /**
+     * @return array<\Spryker\Zed\PublisherExtension\Dependency\Plugin\PublisherPluginInterface>
+     */
+    protected function getStoreStoragePlugins(): array
+    {
+        return [
+            ...
+            new CurrencyStoreWritePublisherPlugin(),
+            ...
+        ];
+    }
+}
+```
+
+
+{% info_block warningBox "Verification" %}
+
+Ensure that, when a story currency added, updated, or deleted, it is exported to or removed from Storage.
+
+Storage type: Redis
+Target entity: Store
+
+Example expected data identifier: `kv:store:de`
+
+Example expected data fragment:
```

Comment:
Is this the data for testing how store currency is updated in Storage?

Reply:
yes. This is an example of the data that should be stored in the storage for the store